### PR TITLE
minor cli docstring fix

### PR DESCRIPTION
**Pull request by jchorl**
Just stumbled upon this, figured I'd fix

### CODE REVIEW

**Review by pditommaso — APPROVED** [review]
This is a good one 😆

**Comment by jchorl** [comment]
> This is a good one 😆

A good _catch_, you might say ;)

File: plugins/nf-cloudcache/src/main/nextflow/cache/CloudCacheFactory.groovy [hunk 1]
```diff
@@ -41,7 +41,7 @@ class CloudCacheFactory extends CacheFactory {
         if( !runName ) throw new AbortOperationException("Missing cache `runName`")
         final path = (Global.session as Session).cloudCachePath
         if( !path )
-            throw new IllegalArgumentException("Cloud-cache path not defined - use either -cloudcatch run option or NXF_CLOUDCACHE_PATH environment variable")
+            throw new IllegalArgumentException("Cloud-cache path not defined - use either -cloudcache run option or NXF_CLOUDCACHE_PATH environment variable")
```
Comment (bentsherman) [inline plugins/nf-cloudcache/src/main/nextflow/cache/CloudCacheFactory.groovy:44]: it should be `-with-cloudcache`

Comment (pditommaso) [reply plugins/nf-cloudcache/src/main/nextflow/cache/CloudCacheFactory.groovy:44]: 🤦

Comment (pditommaso) [reply plugins/nf-cloudcache/src/main/nextflow/cache/CloudCacheFactory.groovy:44]: Done a7f233053

**Pull request merged**